install_dependencies.sh

```diff
@@ -53,11 +53,18 @@ if echo $REPLY | grep -E '^[Yy]$' > /dev/null; then
         if [ -f /etc/lsb-release ] && cat /etc/lsb-release | grep DISTRIB_ID | grep Ubuntu >/dev/null; then
             echo "Installing dependencies (this may take a while)..."
             if sudo apt-get update >/dev/null; then
+                sudo apt-get install --no-install-recommends -y software-properties-common lsb-release
```

Comment:
Should we add lsb-release to the dependencies.md for the sake of completeness?

Reply:
Thought about it but decided against it. We are only using it internally in install_dependencies.sh. If you look into dependencies.md, you probably cannot or don't want to run the install script and are looking for what Hyrise needs. lsb-release is not needed for that.